third_party/move/move-model/src/builder/module_builder.rs

```diff
@@ -114,7 +114,10 @@ pub enum SpecBlockContext<'a> {
     Struct(QualifiedSymbol),
     Function(QualifiedSymbol),
     FunctionCode(QualifiedSymbol, &'a SpecInfo),
-    FunctionCodeV2(QualifiedSymbol, BTreeMap<Symbol, (Loc, Type)>),
+    FunctionCodeV2(
+        QualifiedSymbol,
+        BTreeMap<Symbol, (Loc, Type, Option<TempIndex>)>,
```

Comment:
Option<TempIndex> is necessary to recognize function parameters in the inline spec block